rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/BuildEndpoint.java

```diff
@@ -91,6 +90,10 @@
     static final String BUILD_STATUS = "Status of the build";
 
```

Comment:
Could you add to the `addAttribute` method mention what are the restrictions on the key format?